framework/contrib/ice_updater/include/ErrorLogger.h

```diff
@@ -0,0 +1,84 @@
+/*******************************************************************************
```

Comment:
So.. why do we have a custom error logger here? I noticed parts of the contrib stuff just call mooseError. This seems simpler (that or mooseWarning).